portfolio/src/main/java/com/google/sps/servlets/GameServlet.java

```diff
@@ -21,61 +21,28 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
-
-    Game game = new Game();
-    Gson elements = new Gson();
-
-    @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("application/json;");
-        String json = elements.toJson(game);
-
-        System.out.println(json);
-        response.getWriter().println(json);
-    }
+@WebServlet("/game")
+public class GameServlet extends HttpServlet {
 
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // If the user sends another POST request after the game is over, then start a new game.
+        //gets input from form
+
+        Entity taskEntity = new Entity("Task");
+        taskEntity.setProperty("white", request.getParameter("p1"));
+        taskEntity.setProperty("black", request.getParameter("p2"));
+        taskEntity.setProperty("result", request.getParameter("outcome"));
+        taskEntity.setProperty("moves", request.getParameter("moves"));
```

Comment:
Will a user be sending all moves for a game at once? Or will they be sending each move one at a time. If the former, this is ok. If the latter, then we would end up saving the game once for each move. In that case we might want to pass the game key to update the entity instead of creating a new game each time.

Reply:
Maybe this logic would be nice to wrap in a method in the Game class. Along with a method for converting the entity back to a game (used in the GET).